pkg/kor/formatter.go

```diff
@@ -0,0 +1,209 @@
+package kor
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/olekukonko/tablewriter"
+)
+
+type ResourceInfo struct {
+	Name   string `json:"name"`
+	Reason string `json:"reason,omitempty"`
+}
+
+func unusedResourceFormatter(outputFormat string, outputBuffer bytes.Buffer, opts Opts, jsonResponse []byte) (string, error) {
+	switch outputFormat {
+	case "table":
+		return outputBuffer.String(), nil
```

Comment:
It seems as the Slack conditions were removed, are they no longer needed here?

		if opts.WebhookURL == "" || opts.Channel == "" || opts.Token != "" {
			return outputBuffer.String(), nil
		}
		if err := SendToSlack(SlackMessage{}, opts, outputBuffer.String()); err != nil {
			return "", fmt.Errorf("failed to send message to slack: %w", err)
		}